int32 FAILURE=-3
int32 NOISE=-2
int32 UNCLASSIFIED=-1
int32 CLASSIFIED=0

int32 status
int32 id
float64 x
float64 y 